utils/utils.py

```diff
@@ -305,6 +305,25 @@ def configure_logging(options=None):
         print("Invalid log level (specify: debug, info, warn, error).")
```

Comment:
I noticed that line 302 says `options.get("log", "warn")`.  The `--log` option does not appear to be supported by the current CLI code.  Should it be added, or should this `else` statement be removed?

Reply:
`--log` is still supported by the options for `./gather`, but not for `./scan`. I might have misunderstood where that's called from and am happy to add the option back to the argument handling for `./scan` if that's necessary.

Reply:
I'll defer to @konklone, but I think that would be a useful option for `./scan` too.

Reply:
I don't even remember adding this option, and can't recall ever using it. :) It's not documented [in the README](https://github.com/18F/domain-scan#options), and probably dates way back to the first alpha versions in 2014/2015.

If no one's using it, I'm comfortable removing the option entirely.